bin/cmds/actions.ts

```diff
@@ -1,9 +1,27 @@
 import { loadConfig } from "../../src/config_loader";
 import { fromConfig } from "../../src/actions";
 import { Argv } from "yargs";
+import { errorHandler } from "../../src/error_handler";
 
 // noinspection JSUnusedGlobalSymbols
 export function builder(y: Argv) {
+	return actionsBuilder(y);
+}
+// noinspection JSUnusedGlobalSymbols
+export const command = "actions <action> <group>";
+// noinspection JSUnusedGlobalSymbols
+export const describe = "Run actions on all projects for <action> and <group>";
+// noinspection JSUnusedGlobalSymbols
+export async function handler(argv: any) {
```

Comment:
I believe yargs has support for middleware if you want, but not that it matters too much.